include/miopen/miopen.h

```diff
@@ -7791,6 +7792,47 @@ MIOPEN_EXPORT miopenStatus_t miopenPReLUBackward(miopenHandle_t handle,
 // CLOSEOUT RELU DOXYGEN GROUP
 #endif // MIOPEN_BETA_API
 
+#ifdef MIOPEN_BETA_API
+// adaptiveavgpool APIs
+/** @addtogroup adaptiveavgpool
```

Comment:
Suggestion: `Pooling`
```suggestion
/** @addtogroup Pooling
```